src/main/scala/no/ndla/imageapi/controller/ImageController.scala

```diff
@@ -154,11 +153,22 @@ trait ImageController {
 
       val file = fileParams.getOrElse("file", throw new ValidationException(errors = Seq(ValidationMessage("file", "The request must contain an image file"))))
 
-      writeService.storeNewImage(newImage, file).map(converterService.asApiImageMetaInformationWithApplicationUrl) match {
+      val newImageWithNormalizedLanguages = Try(newImage.copy(
+        titles = newImage.titles.map(t => t.copy(language = LanguageTag(t.language).toString)),
+        alttexts = newImage.alttexts.map(a => a.copy(language = LanguageTag(a.language).toString)),
+        tags = newImage.tags.map(tags => tags.map(t => t.copy(language = LanguageTag(t.language).toString))),
+        captions = newImage.captions.map(captions => captions.map(c => c.copy(language = LanguageTag(c.language).toString)))
```

Comment:
Jeg tror jeg også ville foretrekke å endre domene-objekter til å benytte LanguageTag som type for språk, i stedet for String? På den måten vil man alltid ha en gyldig språkkode gjennom alle lag i applikasjonen.

Da ville man serialisert språkkoden ved lagring til database, og deserialisert den her ved import og ved lesing fra database?

Reply:
Jeg prøver på ny. Og så fjerner jeg NDLA-greiene først i en egen PR.